packages/manager/apps/pci-ai-tools/src/pages/qpus/create/_components/OrderFunnel.component.tsx

```diff
@@ -560,6 +562,49 @@ const OrderFunnel = ({
                 />
               </CardContent>
             </Card>
+            {/* Auto Restart */}
+            <Card
```

Comment:
Issue: This new card really resemble the one in
`packages/manager/apps/pci-ai-tools/src/pages/notebooks/create/_components/OrderSummary.component.tsx`
and
`packages/manager/apps/pci-ai-tools/src/pages/jobs/create/_components/OrderFunnel.component.tsx`

It should be extracted in a component and mutualised.